jsx/LanguageMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import {useEffect, useState} from 'react';
+import {useTranslation} from 'react-i18next';
+import 'i18n';
+
+const supportedLanguageCodes = ['en', 'fr'];
```

Comment:
There is a "language" database table that should be the source of the available languages for a study. I think it would be reasonable to have add a getLanguages to the LORISHelper object created from the smarty template to retrieve it for the menu.

Reply:
I did not bother touching the database / back-end in this PR but this seems like a simple enough change, will do.